mne/datasets/hf_sef/hf_sef.py

```diff
@@ -53,7 +53,7 @@ def data_path(dataset='evoked', path=None, force_update=False,
     destdir = op.join(path, 'HF_SEF')
 
     urls = {'evoked':
-            'https://osf.io/25f8d/download?version=1',
+            'https://zenodo.org/record/3523071/files/hf_sef_evoked.tar.gz',
```

Comment:
I am surprised you don't have to update any hash

Reply:
I am not surprised that I forgot to update the hash...